demo/app/android/app/src/main/java/dev/trustbloc/wallet/MainActivity.kt

```diff
@@ -0,0 +1,105 @@
+package dev.trustbloc.wallet
```

Comment:
The file path includes `java`. Should this be changed ?

Reply:
Its updated